packages/framework/esm-styleguide/src/datepicker/index.tsx

```diff
@@ -232,7 +242,14 @@ const DatePickerInput = forwardRef<HTMLDivElement, DateInputProps>(function Date
       >
         {state.segments.map((segment, i) => cloneElement(props.children(segment), { key: i }))}
       </Group>
-      <Input />
+      <Input
+          type="text"
+          value={state.value ? dayjs(state.value.toDate(tz)).format('MM/DD/YYYY') : ''}
+          onChange={handleInputChange}
+          placeholder="MM/DD/YYYY"
+          className="date-input"
```

Comment:
Since this is a style from the scss file,
```suggestion
          className={styles.dateInput}
```